File: tessoku/B01 - A+B Probrem/B01.js
const main = (input) => {
    const [A, B] = input.trim().split(' ').map(Number);
    const add = (num1, num2) => {
      return num1 + num2;
    }
    console.log(add(A, B));
}
main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: tessoku/B01 - A+B Probrem/B01.cs
using System;

class Program
{
   static int Add(int num1, int num2)
   {
       return num1 + num2;
   }

   static void Main()
   {
       string[] input = Console.ReadLine().Split(' ');
       int A = int.Parse(input[0]);
       int B = int.Parse(input[1]);

       Console.WriteLine(Add(A, B));
   }
}